components/Carousel/Carousel.js

```diff
@@ -1,8 +1,63 @@
 class Carousel {
+    constructor(element) {
+        this.element = element;
 
+        // Get references to the classes we need
+        this.photoBox = this.element.querySelector('.carousel');
+        this.next = this.element.querySelector('.right-button');;
+        this.prev = this.element.querySelector('.left-button');
+        this.photos = this.element.querySelectorAll('.carousel img');
+
+        // Initialize variables
+        this.counter = 0; // Keeps track of which picture is showing
+        this.totalPhotos = this.photos.length;
+        this.current = this.photos[0];
+        console.log(this.totalPhotos)
+
+        // Event listeners for next and previous buttons
+        this.next.addEventListener('click', () => {
+            console.log("Clicked next!")
+            this.navigate(1);
+          });
+          
+        this.prev.addEventListener('click', () => {
+            console.log("Clicked previous!")
+            this.navigate(-1);
+          });
+    }
+
+    navigate(direction) {
+        // Remove the current class from the displayed picture
+        this.current.classList.remove('current');
+
+        // Keeps track of which picture is currently showing
+        this.counter = this.counter + direction;
+        console.log(this.counter)
+
+        // Take care of edge cases
+        if (direction === -1 && this.counter < 0) { 
+          this.counter = this.totalPhotos - 1; // If on the first picture and previous is clicked
+        }
+        if (direction === 1 && !this.photos[this.counter]) { 
+          this.counter = 0; // If at the end of the carousel and next picture is clicked
+        }
+
+        console.log(this.counter)
+
+        // Set the picture to current
+        this.current = this.photos[this.counter];
+
+        // Add the current class to the displayed picture
+        this.current.classList.add('current');
+      }
 }
 
-let carousel = document.querySelector();
+let carousel = document.querySelector(".carousel");
+
+carousel = new Carousel(carousel)
+carousel.navigate(1);
+
+
```

Comment:
Again make sure to clean up your console logs.
Great practice on commenting your code! 👍